docker/geoserver/Dockerfile

```diff
@@ -4,32 +4,33 @@ LABEL GeoNode Development Team
 #
 # Set GeoServer version and data directory
 #
-ENV GEOSERVER_VERSION=2.18.2
+ENV GEOSERVER_VERSION=2.19.x
 ENV GEOSERVER_DATA_DIR="/geoserver_data/data"
 
 #
 # Download and install GeoServer
 #
 RUN cd /usr/local/tomcat/webapps \
-    && wget --no-check-certificate --progress=bar:force:noscroll https://www.dropbox.com/s/xlich7pmneaupqp/geoserver-${GEOSERVER_VERSION}.war?dl=1 -O geoserver-${GEOSERVER_VERSION}.war \
-    && unzip -q geoserver-${GEOSERVER_VERSION}.war -d geoserver \
-    && rm geoserver-${GEOSERVER_VERSION}.war \
+    && wget --no-check-certificate --progress=bar:force:noscroll https://artifacts.geonode.org/geoserver/${GEOSERVER_VERSION}/geoserver.war -O geoserver.war \
+    && unzip -q geoserver.war -d geoserver \
+    && rm geoserver.war \
     && mkdir -p $GEOSERVER_DATA_DIR
 
 VOLUME $GEOSERVER_DATA_DIR
 
 # added by simonelanucara https://github.com/simonelanucara
 # Optionally add JAI, ImageIO and Marlin Render for improved Geoserver performance
 WORKDIR /tmp
-ARG JAI_IMAGEIO=true
-RUN if [ "$JAI_IMAGEIO" = true ]; then \
-    # marlin plugin
-    wget https://github.com/bourgesl/marlin-renderer/releases/download/v0_9_3/marlin-0.9.3-Unsafe.jar && \
-    mv /tmp/marlin-0.9.3-Unsafe.jar /usr/local/tomcat/webapps/geoserver/WEB-INF/lib/ ; \
-    fi
+ 
+# ARG JAI_IMAGEIO=true
+# RUN if [ "$JAI_IMAGEIO" = true ]; then \
+#     # marlin plugin
+#     wget https://github.com/bourgesl/marlin-renderer/releases/download/v0_9_3/marlin-0.9.3-Unsafe.jar && \
+#     mv /tmp/marlin-0.9.3-Unsafe.jar /usr/local/tomcat/webapps/geoserver/WEB-INF/lib/ ; \
+#     fi
```

Comment:
Let's delete this outdated stuff instead of commenting it.

Reply:
I left it there because there is a still unreported problem I found today to resolve: the geoserver 2.19.x iI build is not loading marlin at all, the marlin jar is already provided by geoserver, as it should for newer versions, @randomorder told me to leave this in the end of my upgrading work

Reply:
removed commented code

Reply:
So the marlin renderer is already implemented as of GeoServer 2.8.2 ++? What about the more recent releases of marlin renderer like 9.4.x ? Will they work with the latest GeoServer?